fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -118,13 +135,20 @@ public boolean storeWholePermission() {
     try {
       view = fiatService.getUserPermission(username);
     } catch (RetrofitError re) {
-      String message = String.format("Cannot get whole user permission for user %s", username);
-      log.debug(message);
-      log.trace(message, re);
+      String message = String.format("Cannot get whole user permission for user %s. Cause: %s",
+                                     username,
+                                     re.getMessage());
+      if (log.isDebugEnabled()) {
+        log.debug(message, re);
```

Comment:
The ability to tell users to turn on debug logging and getting everything instead of debug for some, and trace for others.